resources/views/listing/filters.blade.php

```diff
@@ -1,18 +1,15 @@
 @php($id = uniqid('filters-'))
 <x-rapidez::slideover.mobile :$id :title="__('Filters')" position="right">
     <div class="w-full p-2 max-lg:bg-white max-lg:p-5">
-        {{-- On mobile the filters aren't immedately visible so we should defer loading --}}
-        <lazy>
```

Comment:
Does this cause any impact on mobile performance?

Reply:
To a degree, for sure, however I do not believe there is much we can do about that in the current state. It seems a limitation of instantsearch that the filter widgets need to be loaded in immediately if you want your routing parameters to work. Note that the actual filtering of the results comes directly from the filter widgets.

Supposedly, a workaround would be to mount virtual widgets. I could not get this to work properly and I can only imagine this making performance worse across the board.

Something we could also do is only lazy load the filters if there are no url parameters. I don't know if I necessarily like this approach but it would work...